crates/egui/src/widgets/text_edit/builder.rs

```diff
@@ -978,10 +978,14 @@ fn events(
             Event::Key {
                 key: Key::Z,
                 pressed: true,
-                modifiers,
+                modifiers:
+                    Modifiers {
+                        command: true,
+                        shift: false,
+                        ..
+                    },
                 ..
-            } if modifiers.command && !modifiers.shift => {
-                // TODO(emilk): redo
+            } => {
```

Comment:
This is a bad way to match against Modifiers, as this will not catch `Ctrl+Z` on Windows and Linux, because that also sets the `ctrl` fields.

The old way was not great either, because it would ignore the `alt` key.

The preferred way of test `Modifiers` is to use `Modifiers::matches`:

It is better to use `if modifiers.matches(Modifiers::COMMAND)`

Reply:
My apologies, that method was not being used here so I wasn't aware of its existence. I'll fix this up

Reply:
No need to apologize - I didn't mean to come off as harsh. This is not easy to get right unless you already know it :)